compiler/optimizer/OMRValuePropagation.hpp

```diff
@@ -90,6 +90,9 @@ template <class T> class TR_Stack;
 typedef TR::Node* (* ValuePropagationPtr)(OMR::ValuePropagation *, TR::Node *);
 extern const ValuePropagationPtr constraintHandlers[];
 
+typedef TR::typed_allocator<std::pair<TR::CFGEdge *, TR_BitVector*>, TR::Region &> DefinedOnAllPathsMapAllocator;
```

Comment:
```suggestion
typedef TR::typed_allocator<std::pair<TR::CFGEdge * const, TR_BitVector*>, TR::Region &> DefinedOnAllPathsMapAllocator;
```

I believe this will fix the build failure.

Reply:
Thanks @Leonardo2718 I have updated my commit - sorry I have been away